### DISCUSSION TITLE
Flashing

### DISCUSSION THREAD

**Discussion by Greginkansas** [toverainc/willow#220] [General]
Can I use the  willow-dist-ESP32_S3_BOX.bin  to flash willow now?

**Comment by hamishcunningham** [comment]
Hi! When you've followed the instructions here https://github.com/toverainc/willow/#getting-started
and got to the "./utils.sh build" stage from within docker, you can then quit the container and do "./utils.sh flash".

There are other ways, including taking the .bin and doing it yourself, but these are quite error prone and complex.

With the upcoming 1.0 release there will be an easier way, but for the time being you need to follow the instructions linked above.

HTH!

**Reply by winghavenrc** [reply]
When's the ETA for 1.0?

**Reply by tombloemker** [reply]
Mid July is the only data I've seen

**Reply by kristiankielhofner** [reply]
We're still targeting this date but we've identified a top-priority show stopping issue where the device fails to wake after several hundred sessions after boot. Ah the challenges of a microcontroller in C...

I typically do anything possible to make deliverables but we're absolutely not going to sacrifice stability - especially for a stable release!

**Reply by winghavenrc** [reply]
Makes sense

**Reply by Takore2002** [reply]
Is it going to be a joint 1.0 release for Willow and WIS?

**Reply by kristiankielhofner** [reply]
We'll be using semantic versioning and it's an early stable release so it will likely be a joint 0.x
